x/stakeibc/keeper/icqcallbacks_validator_test.go

```diff
@@ -0,0 +1,205 @@
+package keeper_test
+
+import (
+	ibctesting "github.com/cosmos/ibc-go/v3/testing"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	stakingtypes "github.com/cosmos/cosmos-sdk/x/staking/types"
+
+	epochtypes "github.com/Stride-Labs/stride/x/epochs/types"
+	icqtypes "github.com/Stride-Labs/stride/x/interchainquery/types"
+	stakeibckeeper "github.com/Stride-Labs/stride/x/stakeibc/keeper"
+	stakeibctypes "github.com/Stride-Labs/stride/x/stakeibc/types"
+)
+
+type ValidatorICQCallbackState struct {
+	hostZone           stakeibctypes.HostZone
+	strideEpochTracker stakeibctypes.EpochTracker
+}
+
+type ValidatorICQCallbackArgs struct {
+	query        icqtypes.Query
+	callbackArgs []byte
+}
+
+type ValidatorICQCallbackTestCase struct {
+	initialState         ValidatorICQCallbackState
+	validArgs            ValidatorICQCallbackArgs
+	valAddress           string
```

Comment:
nit: this is unused, either remove or use tx.ValAddress in ln 47

Reply:
removed!